# clusters_into_profiles

# Naming the clusters!

# Cluster 0

**Numeric profile**

High Positive PC1 → strong profeisonal oriented users

Slightly Positive PC2 → some level of interaccion with the social network

Neutral PC3 → average usage frequency

**Conceptual Interpretation**

Students who use LinkedIn primarily as a professional and career tool, with moderate but not intensive interaction.

**Suggested Name:**
# Career-Oriented Student Users

# Cluster 1

**Numeric profile**

High Negative PC1 → weak profeisonal oriented users

Slightly Positive PC2 → some level of interaccion with the social network

Slightly Positive PC3 → some level of usage frequency

**Conceptual Interpretation**

Students who use LinkedIn interactively and frequently, but without a clear focus on career or employment.

**Suggested Name:**
# Socially Active Explorers

# Cluster 2

**Numeric profile**

Slightly Negativ PC1  → weak profeisonal oriented users

Slightly Negative PC2 → passive interaccion with the social network

Slightly Negative PC3 → low level of usage frequency

**Conceptual Interpretation**

Students with limited, passive, and infrequent use of LinkedIn.

**Suggested Name:**
# Low-Engagement Passive Users

It would be nice to see to what level/frequency each cluster does things according to the questions made

In [59]:
df.groupby('cluster')[latent_features].mean().T.round(2)

cluster 
 0 
 1 
 2 
 
 
 
 
 Q17_LinkedIn_Instructors 
 4.50 
 3.18 
 3.82 
 
 
 Q17_LinkedIn_Classmates 
 4.50 
 3.27 
 3.92 
 
 
 Q17_LinkedIn_Career 
 5.00 
 3.73 
 4.60 
 
 
 Q17_LinkedIn_Exposure 
 4.39 
 3.45 
 3.92 
 
 
 Q17_LinkedIn_Expression 
 4.11 
 3.18 
 3.45 
 
 
 Q17_LinkedIn_Dissemination 
 4.50 
 3.45 
 3.68 
 
 
 Q18_LinkedIn_Interact 
 4.83 
 3.55 
 4.40 
 
 
 Q18_LinkedIn_Entertained 
 3.50 
 2.45 
 2.85 
 
 
 Q18_LinkedIn_Informed 
 4.50 
 3.45 
 3.98 
 
 
 Q18_LinkedIn_Career 
 5.00 
 3.64 
 4.58 
 
 
 Q18_LinkedIn_Professional 
 5.00 
 3.73 
 4.40 
 
 
 Q20_LinkedIn_UseProf 
 4.89 
 3.64 
 4.18 
 
 
 Q20_LinkedIn_UseJob 
 4.78 
 2.82 
 4.05 
 
 
 Q20_LinkedIn_UseCareer 
 4.89 
 3.00 
 4.20 
 
 
 Q20_LinkedIn_UseNetwork 
 4.94 
 3.36 
 4.15 
 
 
 Q20_LinkedIn_UseContacts 
 4.78 
 3.18 
 3.82 
 
 
 Q20_LinkedIn_UseValue 
 4.89 
 3.00 
 4.10 
 
 
 Q22_LinkedIn_FreqYear 
 3.11 
 2.00 
 2.82 
 
 
 Q23_LinkedIn_FreqSenior 
 3.78 
 2.73 
 3.32 
 
 
 Q24_LinkedIn_FreqCareer 
 3.78 
 2.09 
 3.22

In [60]:
# Rate of students by cluster
df.groupby('cluster').mean()

# Academic Area
df.groupby('cluster')[[
    'is_business',
    'is_arts & sciences',
    'is_health professions'
]].mean().round(2)

is_business 
 is_arts & sciences 
 is_health professions 
 
 
 cluster 
 
 
 
 
 
 
 
 0 
 0.61 
 0.17 
 0.22 
 
 
 1 
 0.27 
 0.45 
 0.27 
 
 
 2 
 0.38 
 0.28 
 0.32

The distribution of academic areas across clusters revealed meaningful differences. The career-oriented cluster was predominantly composed of business students, whereas the exploratory cluster showed a higher proportion of arts and sciences students. In contrast, the passive professional cluster displayed a more balanced academic composition, suggesting that passive recognition of LinkedIn’s professional value transcends disciplinary boundaries

# **Finally Let's Convert Each Cluster Into Persons (Student)**

🧠 Student 1 – “Strategic Career Builder”

📍 Cluster 0

***General Profile***

A student who uses LinkedIn intentionally, strategically, and professionally, integrating the platform into their career planning, networking, and professional identity development.

According to the results:

High scores in:

Profesional and Career Usage (Q20_* ≈ 4.8–5.0)

Active interaction and profesional value (Q18_* ≈ 4.5–5.0)

Connecting with teachers, industry, and peers (Q17_* ≈ 4.1–5.0)

Moderate-high frequency of use

61% business students

Higher proportion of seniors students

***Motivations***

Building professional capital

Strategic networking

Preparation for job placement

***Representative phrase***

“I use LinkedIn as a key tool to build my career.”

***Practical implications***

High receptiveness to:

Mentoring

Recruitment events

Employer branding content

Ideal for university employability programs

🌱 Student 2 – “Exploratory Observer”

📍 Cluster 1

***General Profile***

Students with low professional engagement on LinkedIn. Their use is sporadic, exploratory, and poorly integrated with academic or career goals.

According to the results:

Lower scores across all professional dimensions (≈ 2.5–3.6)

Less frequent use

Predominance of arts & sciences students

Lower presence of seniors

***Motivations***

General curiosity

Passive observation of the professional environment

Lack of career clarity

***Representative Phrase***

“I know LinkedIn exists, but I don’t use it much, and I don’t really know what for.”

***Practical Implications***

They need:

Digital professional literacy

Clear examples of usefulness

Guided curriculum integration

They do not respond well to highly professional messages without context

👀 Student 3 – “Passive Professional”

📍 Cluster 2

General Profile

Students who recognize the professional value of LinkedIn but maintain a primarily passive use, focused on consuming information rather than interacting or networking.

According to the results:

Medium-high scores in professional value (≈ 4.0–4.6)

Lower active interaction (Q18_LinkedIn_Interact ≈ 4.4, but lower than cluster 0)

Moderate frequency of use

Balanced academic distribution

***Motivations***

Staying informed

Monitoring opportunities

Preparing for the future

***Representative Phrase***

“LinkedIn is important, although I don’t use it actively yet.”

***Practical Implications***

High activation potential:

Practical workshops

Incentives for interaction (posts, comments)

Relevant success stories

Ideal profile for transition interventions